include/zephyr/bluetooth/bluetooth.h

```diff
@@ -682,7 +682,7 @@ struct bt_le_local_features {
 int bt_le_get_local_features(struct bt_le_local_features *local_features);
 
 /** Advertising options */
-enum advertising_options {
+enum bt_le_adv_opt {
```

Comment:
I'm a bit worried that someone has defined a `enum advertising_opions` as a local variable, which would make this a stable API breaking change :|

Reply:
The change came in last week, and I don't suppose there's been a release since then? 
https://github.com/zephyrproject-rtos/zephyr/commit/2ae411017d9e753b263e3e875ae2c11583154769

Of course if it's a problem I can revert it, just thought it'd be nice to align the names :)

Reply:
Ah, in that case it's a non-issue :) Thanks